wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -1071,6 +1080,7 @@ public final class SessionManager: NSObject, SessionManagerType {
             for: account,
             coreDataStack: coreDataStack,
             configuration: sessionConfig,
+            analyticsSessionConfiguration: analyticsSessionConfiguration,
```

Comment:
Why is it required to pass the configuration through several layers? Because of unit tests or something?
If possible I would create it on the lowest possible level, for me it looks like `ZMUserSession.setupAnalyticsSession`

Reply:
I don't have access to the countly key and host in SE, but on UI project. Therefore I need to pass it all the way up to the UI Project app delegate.